content/doc/tutorials/creating-a-pipeline-in-blue-ocean.adoc

```diff
@@ -354,9 +365,19 @@ Blue Ocean interface with all previous runs displayed",width=100%]
 Well done! You've just used the Blue Ocean feature of Jenkins to build a simple
 Node.js and React application with npm!
 
-The "build", "test" and "deliver" stages you created above are the basis for
-building more complex Node.js and React applications in Jenkins, as well as
-Node.js and React applications that integrate with other technology stacks.
+Notice where Blue Ocean placed the
+link:/doc/book/pipeline/syntax#environment[`environment`] directive in its
+Jenkinsfile. This directive's location within the "Test" stage means that the
+environment variable `CI` (with its value of `true`) is only available within
+the scope of this "Test" stage. How might you set this directive in Blue Ocean
+so that its environment variable is available globally throughout Pipeline (as
```

Comment:
Do you answer this question in this tutorial?

Reply:
No - but I think you're hinting at (as I'm agreeing with you) that this needs spelling out.

I'll write this up in a separate *Follow Up* section above *Wrapping Up*, this this content will now deserve its own subsection.

Reply:
Done :-)

Reply:
Oh - let me know if you're not a fan of rhetorical questions ... We can change some wording 😉

Reply:
Okay - I've removed the rhetorical question.